framework/src/base/ComputeUserObjectsThread.C

```diff
@@ -218,29 +113,25 @@ ComputeUserObjectsThread::onInternalSide(const Elem *elem, unsigned int side)
     elem_id = elem->id(),
     neighbor_id = neighbor->id();
 
-  // Only perform execute if there are objects
-  if (global_uo.size() > 0 || block_uo.size() > 0)
```

Comment:
This was here to shortcut this whole block if we didn't have any UOs. It wasn't necessary but perhaps useful. Are we hitting this loop now even when we don't have objects? Perhaps it's just reversed. I see a `hasActiveBlockObjects()` just below but that might be cheaper than all this neighbor logic.